source/stylesheets/partials/_get_started.css.scss

```diff
@@ -48,10 +48,4 @@ section.get-started {
       text-align: left;
     }
   }
-
-  pre {
```

Comment:
Did we borrow this from `asciicinema-player.css` previously, and inlining it makes this rule redundant?

Reply:
was introduced here https://github.com/rspec/rspec.github.io/commit/cd123553541b9529a4810a2c87ebcfd09bc059b2#diff-09a747412602d0fdaf667a1cab874c6e
but was not used
it's broke player layout, asciinema js script adds <pre> element around player, so I dropped it to fix that